server.js

```diff
@@ -3,7 +3,7 @@ const express = require('express');
 const app = express();
 
 app.get('/ping', (req, res) => {
-    res.send('pong')
+    res.json({'message': 'pong'})
```

Comment:
Consider using a constant for the 'message' field in the response JSON object to avoid hardcoding strings. [medium]